### PR TITLE
Ndcs country, full and search grid

### PR DESCRIPTION
**Pull request by j8seangel**
This PR includes the new grid system for all of the ndcs pages:
- Index: http://localhost:3000/ndcs
- Country: http://localhost:3000/ndcs/country/CAN
- Compare: http://localhost:3000/ndcs/compare/mitigation?locations=CAN
- Full: http://localhost:3000/ndcs/country/AFG/full
- Search: http://localhost:3000/ndc-search?searchBy=query&query=forest

There are still some issues to fix:
- Full: sticky empty space and intro
- Search: fix map when sticky and result list background

### CODE REVIEW

**Comment by Bluesmile82** [comment]
image
The Anchor nav is not aligned in the search page

**Comment by j8seangel** [comment]
I've found a small issue on the search results page on the responsive version
image
The `resultsCard` is giving it a different padding, could you review it?

I can't select this selector
image

Please review the hover styles of the selectors too.

[This page](https://climate-watch.vizzuality.com/ndcs/country/USA/full?query=7&searchBy=goal) is still using the grid and seems to be misaligned
image

**Comment by weberjavi** [comment]
Ready to revision 🙏

File: app/javascript/app/styles/layout.scss [hunk 1]
```diff
@@ -78,55 +84,10 @@ $grid-padding-gutters: $gutter-padding;
...
-  @media #{$desktop} {
-    padding-top: 0;
-  }
-}
-
 .content {
   @include row(12);
 }
 
-%grid {
```
Comment (j8seangel) [inline app/javascript/app/styles/layout.scss]: 👏

**Pull request merged**